nox/sessions.py

```diff
@@ -177,6 +177,10 @@ def run(self, *args, env=None, **kwargs):
         if not args:
             raise ValueError("At least one argument required to run().")
 
+        if self._runner.global_config.install_only:
+            logger.info("Skipping {} run, as --install-only is set.".format(args[0]))
+            return
```

Comment:
Maybe is better to raise a custom exception? 

Something like https://github.com/theacodes/nox/blob/c05f131c3b7255bde09cf597c93fc97d0f1ab5c2/nox/sessions.py#L59-L60

Reply:
Nah, this is fine.

Reply:
I think it's desirable to skip `run` commands rather than terminating when encountering them. That way, Nox will install all dependencies in a session of the form

```python
session.install('spam')
session.run('spam', ...)
session.install('eggs')
session.run('eggs', ...)
```

This will likely be more common when the second install/run pair is in another session invoked by the first with `notify`, as Nox's own noxfile does here:

https://github.com/theacodes/nox/blob/c05f131c3b7255bde09cf597c93fc97d0f1ab5c2/noxfile.py#L32

So, when I run `nox -s tests --install-only` in the root of this repo, "coverage" should be installed.